src/relay/ir/alpha_equal.cc

```diff
@@ -60,6 +60,19 @@ class AlphaEqualHandler:
       if (!rhs->derived_from<ExprNode>()) return false;
       return ExprEqual(Downcast<Expr>(lhs), Downcast<Expr>(rhs));
     }
+    if (const auto lhsm = lhs.as<ModuleNode>()) {
```

Comment:
Why are we alpha-equaling the whole module?

cc @zhiics

Reply:
I don't think I made this change. And I don't think we should do that as well.

Reply:
I need this to test roundtrip on whole module.
Also there is no reason why we dont want equality test on module (we can use this to test pure Module -> Module pass work as intended).

Reply:
Yeah, I have the same question here. Do we really need to compare the whole module?

Reply:
It sounds reasonable to me to have it for testing, although we don't have comparison of modules with multiple functions in most test cases.